managed/src/main/java/com/yugabyte/yw/controllers/UniverseController.java

```diff
@@ -128,6 +127,22 @@ public Result findByName_Corrected(UUID customerUUID, String universeName) {
     return Results.status(OK, Json.toJson(universe.universeUUID));
   }
 
+  /**
```

Comment:
Update PR description to explain why findByName_corrected cannot be used by UI.
Looking up resource (find) is a commonly exposed API and not verifyUnique.

Reply:
Added